torch/_inductor/compile_fx.py

```diff
@@ -2295,10 +2295,17 @@ def partition_fn(
                 "static_lifetime_input_indices", None
             )
 
+            # TODO(xuanzh): do I need to change the first argument of dynamo_timed below?
```

Comment:
yeah, i think it would be nice if customize the dynamo_timed message below to print something like f`custom_partitioner_pass: {type(custom_partition_fn)}` so it's a bit easier to tell from tlparse if or what custom partitioner someone is using in their job